src/TodoListItem.jsx

```diff
@@ -1,7 +1,10 @@
-function TodoListItem({title}) {
+function TodoListItem({todo, onCompleteTodo}) {
   return (
     <li>
-      {title}
+      <form>
+        <input type = "checkbox" checked = {todo.isComplete} onChange = {() => onCompleteTodo(todo.id)}></input>
```

Comment:
I would remove spaces surrounding the equal sign. Usually, it's written without spaces.